ledger/sandbox/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think there should be a shared `logback.xml` that can be used.

Reply:
Good call.

Reply:
Done.